src/Crypto/WebAuthn/Internal/ToJSONOrphans.hs

```diff
@@ -23,8 +23,14 @@ import qualified Data.Text.Encoding as Text
 import qualified Data.X509 as X509
 import qualified Data.X509.Validation as X509
 
-instance ToJSON BS.ByteString where
-  toJSON = String . Text.decodeUtf8 . Base16.encode
+newtype Base16ByteString = Base16ByteString BS.ByteString
+  deriving newtype (Eq)
+
+instance ToJSON Base16ByteString where
+  toJSON (Base16ByteString bytes) = String . Text.decodeUtf8 . Base16.encode $ bytes
+
+instance Show Base16ByteString where
+  show (Base16ByteString bytes) = Text.unpack . Text.decodeUtf8 . Base16.encode $ bytes
```

Comment:
This type might need a better name. It's not that it contains a base16 bytestring, but rather it's the same as ByteString but with `ToJSON` and `Show` instances that print it as base16 for debugging purposes. This should be in this types documentation. How about calling it `PrintableByteString` instead.